.travis.yml

```diff
@@ -41,23 +41,24 @@ jobs:
         # Make sure README is up to date
         - diff --ignore-trailing-space <(sed -n '/^driver /,/^```/p' README.md | head -n -1) <(cargo run --bin driver -- --help)
         # Build image with compiled binary
-        - docker build --tag stablex-binary-public --build-arg SOLVER_BASE=gnosispm/dex-open-solver:$OPEN_SOLVER_VERSION -f driver/docker/rust/Dockerfile .
+        - docker build --tag stablex-binary-public --build-arg SOLVER_BASE=gnosispm/dex-open-solver:$OPEN_SOLVER_VERSION --build-arg ECONOMIC_VIABILITY_STRATEGY=static --build-arg STATIC_MIN_AVG_FEE_PER_ORDER=0 --build-arg STATIC_MAX_GAS_PRICE=100000000000 -f driver/docker/rust/Dockerfile .
```

Comment:
I dont' think the extra build args do anything here. Those are args used in Dockerfiles I don't expect them to be passed to the driver.
We pass them through the env file specified in docker-compose.yml, so by adding the new env file there for the service stablex-test you have changed them.

Reply:
They certainly do not. I thought these had been removed.

Reply:
They are gone now.